crates/subspace-node/src/secondary_chain/chain_spec.rs

```diff
@@ -121,10 +121,15 @@ pub fn x_net_config() -> ExecutionChainSpec<GenesisConfig> {
                         .expect("Wrong root account address"),
                 ],
                 vec![(
-                    get_account_id_from_seed("Alice"),
+                    AccountId::from_ss58check("5CXTmJEusve5ixyJufqHThmy4qUrrm6FyLCR7QfE4bbyMTNC")
+                        .expect("Wrong root account address"),
                     1_000 * SSC,
-                    get_account_id_from_seed("Alice"),
-                    get_public_key_from_seed::<ExecutorPublicKey>("Alice"),
+                    AccountId::from_ss58check("5CXTmJEusve5ixyJufqHThmy4qUrrm6FyLCR7QfE4bbyMTNC")
```

Comment:
What about using some other keys than out "production" testnet keys for these networks and certainly for executor?

Reply:
I thought you might want to have fewer pre-configured accounts :), yeah, I also prefer it that way, we need to give this genesis executor some initial balance same with the root account then.

Reply:
Force pushed with a dedicated executor account specified in the secondary chain spec.